pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "baseservice"
+version = "0.0.1a"
+requires-python = '>=3.7,<3.11'
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+]
+dynamic = [
+    "dependencies",
+    "optional-dependencies"
+]
+
+[project.urls]
+github-repo = "https://github.com/Avivsalem/BaseService"
+
+[[project.authors]]
+name = "Aviv Salem"
+email = "avivsalem@gmail.com"
+
+[[project.maintainers]]
+name = "Aviv Salem"
+email = "avivsalem@gmail.com"
+
+[[project.maintainers]]
+name = "Nir Schulman"
+email = "narsssx@gmail.com"
+
+[tool.setuptools]
+packages = ["baseservice"]
+platforms = [
+    "win32",
+    "linux"
+]
+
+[tool.setuptools.dynamic]
```

Comment:
this is the 'extra requires'? 
what about 'all'? is there a standard way to do this?